### PR TITLE
Skip this side check if fluid conduit not connected.

### PR DESCRIPTION
**Pull request by lordIcocain**
Fixes: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/18130

### CODE REVIEW

File: src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java [hunk 1]
```diff
@@ -274,15 +276,16 @@ public boolean containsItems() {
...
-                    && !isItemConduit(this.posInterface.getOffSet(dir).getTileEntity())) {
-                final int result = checkItemFluids(this.getSideFluid(dir), this.getSideItem(dir), dir.getOpposite());
-                if (result == 1) {
-                    return true;
-                }
-                if (result != 2) {
-                    anyValid = true;
-                }
+            TileEntity te = this.posInterface.getOffSet(dir).getTileEntity();
+            if (te == null || !checkValidSide(te, dir)
```
Comment (Cardinalstars) [inline src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java:280]: Please make this logic a bit simplier or break it into multiple checks or make it more readable with parenthesis placement. Without extra parenthesis the second line is ambiguious to the reader.

File: src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java [hunk 2]
```diff
@@ -274,15 +276,16 @@ public boolean containsItems() {
...
+                    || (isConduit(te) && (isItemConduit(te) || !isFluidConduitConnected(te, dir))))
+                continue;
+            final int result = checkItemFluids(this.getSideFluid(dir), this.getSideItem(dir), dir.getOpposite());
```
Comment (Cardinalstars) [inline src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java:283]: Generally, I woud prefer if we didnt use continue.

File: src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java [hunk 3]
```diff
@@ -356,6 +359,15 @@ private boolean isGTMachine(Object o) {
         return false;
     }
 
+    private boolean isFluidConduitConnected(TileEntity te, ForgeDirection dir) {
+        try {
+            ILiquidConduit liquidConduit = (ILiquidConduit) eioTypeCheck.invoke(te, ILiquidConduit.class);
```
Comment (Cardinalstars) [inline src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java:364]: Is this really how this is normally used? Seems really strange to be catching multiple exceptions like that. I will look at this more.

**Review by Cardinalstars — CHANGES_REQUESTED** [review]

File: src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java (hunk 2 above)
Comment (Lyfts) [reply src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java:283]: What's wrong with `continue` now? It's used incredibly frequently in java

File: src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java (hunk 1 above)
Comment (lordIcocain) [reply src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java]: I changed logic, hope better.

File: src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java (hunk 2 above)
Comment (lordIcocain) [reply src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java]: no continue anymore

**Comment by serenibyss** [comment]
@Cardinalstars Could you take another look at this? Or dismiss your previous review if you're unavailable and I can take a look

**Review by Cardinalstars — APPROVED** [review]

File: src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java (hunk 3 above)
Comment (Cardinalstars) [reply src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java:368]: This seems fine. Just strange.

**Pull request merged**